rdr_service/model/genomics.py

```diff
@@ -1781,3 +1781,48 @@ class GenomicP0Raw(Base):
     validation_passed = Column(String(255), nullable=True)
     ai_an = Column(String(255), nullable=True)
     p_site_id = Column(String(255), nullable=True)
+
+
+class GenomicP1Raw(Base):
```

Comment:
I don't see an alembic migration for this model. Should that be included?

Reply:
Good catch, forgot to commit it.